src/Linux/Native/RunAsUser/src/main.cpp

```diff
@@ -294,13 +297,15 @@ int run_rhost(const picojson::object& json, const char* user, const gid_t gid, c
             if (err) {
                 logf(log_verbosity::minimal, "Error rhost exited:[%d] %s\n", err, strerror(err));
             } else {
-                logf(log_verbosity::minimal, "RHost exited.\n");
+                logf(log_verbosity::minimal, "RHost exited normally.\n");
             }
         } else if (WIFSIGNALED(ws)) {
             logf(log_verbosity::minimal, "Error rhost terminated by a signal: %d\n", WTERMSIG(ws));
```

Comment:
Not related to this change, but in log messages accessible to the user I'd put `Microsoft.R.Host.exe` rather than `rhost` since no one out there knows what `rhost` means